lib/ansible/plugins/filter/core.py

```diff
@@ -260,48 +261,56 @@ def get_hash(data, hashtype='sha1'):
     try:  # see if hash is supported
         h = hashlib.new(hashtype)
     except:
-        return None
+        raise errors.AnsibleFilterError('|hash requires hashlib python module')
```

Comment:
Maybe just tell: this algo is not supported | except before if hashlib is not installed